src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -2392,6 +2391,32 @@ bool CacheManager::ResetPT(const std::string& file_name) {
   return result;
 }
 
+policy::RequestType::State CacheManager::GetAppRequestTypesState(
+    const std::string& policy_app_id) const {
+  LOG4CXX_AUTO_TRACE(logger_);
+  sync_primitives::AutoLock auto_lock(cache_lock_);
+  policy_table::ApplicationPolicies::iterator app_policies_iter =
```

Comment:
@AByzhynar It would be better use const_iterator, as I can see iterator doesn't changing.

Reply:
@KVGrygoriev Done in 8edb44a0fe04914980fa2afd9719b2a6fdcb79f0